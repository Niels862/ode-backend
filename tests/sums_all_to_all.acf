chip {
    cabs: [
        cab 1 with clocks 1, - {
            cams: [
                SumInv as sum1 {
                    inputs: 3,
                },
            ],
        },
        cab 2 with clocks 1, - {
            cams: [
                SumInv as sum2 {
                    inputs: 3,
                },
            ],
        },
        cab 3 with clocks 1, - {
            cams: [  
                SumInv as sum3 {
                    inputs: 3,
                },
            ],
        },
        cab 4 with clocks 1, - {
            cams: [
                SumInv as sum4 {
                    inputs: 3,
                },
            ],
        },
    ],
    routing: [
        sum1 -> sum2:1,
        sum1 -> sum3:1,
        sum1 -> sum4:1,

        sum2 -> sum1:1,
        sum2 -> sum3:2,
        sum2 -> sum4:2,

        sum3 -> sum1:2,
        sum3 -> sum2:2,
        sum3 -> sum4:3,

        sum4 -> sum1:3,
        sum4 -> sum2:3,
        sum4 -> sum3:3,
    ],
}