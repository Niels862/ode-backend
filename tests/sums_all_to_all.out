0 0 0 0 32 4 0 2 5 0 0 64 0 0 81 255 15 241 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 64 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 2 255 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 255 255 255 255 255 255 255 255 0 0 0 0 0 0 0 0 1 19 1 129 1 29 1 129 1 27 1 129 1 31 1 129 12 0 0 0 0 0 0 0 0 0 0 0 0 48 0 16 0 208 0 16 0 0 0 176 0 16 0 5 0 240 0 16 255 255 255 255 255 255 255 255 0 0 0 0 0 0 0 0 1 19 1 129 1 31 1 129 1 29 1 129 1 27 1 129 12 0 0 0 0 0 0 0 0 0 0 0 0 48 0 16 0 240 0 16 0 0 0 208 0 16 0 5 0 176 0 16 255 255 255 255 255 255 255 255 0 0 0 0 0 0 0 0 1 19 1 129 1 31 1 129 1 25 1 129 1 27 1 129 12 0 0 0 0 0 0 0 0 0 0 0 0 48 0 16 0 240 0 16 0 0 0 144 0 16 0 5 0 176 0 16 255 255 255 255 255 255 255 255 0 0 0 0 0 0 0 0 1 19 1 129 1 29 1 129 1 25 1 129 1 27 1 129 12 0 0 0 0 0 0 0 0 0 0 0 0 48 0 16 0 208 0 16 0 0 0 144 0 16 0 5 0 176 0 16